services/QuillLMS/app/controllers/quill_big_query/generated_views_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module QuillBigQuery
-  class MaterializedViewsController < ApplicationController
+  class GeneratedViewsController < ApplicationController
```

Comment:
The old nomenclature described an implementation detail (mat views), which led to a lot of code diffs here that are only name changes. 

+1 for moving to a naming scheme that describes the function, rather than the implementation